code/datums/diseases/advance/symptoms/weakness.dm

```diff
@@ -0,0 +1,44 @@
+/*
+//////////////////////////////////////
+
+Weakness
+
+	Very very noticable.
+	Lowers resistance.
+	Decreases stage speed.
+	Decreases transmittablity tremendously.
+	Moderate Level.
+
+Bonus
+	Deals stamina damage to the host
+
+//////////////////////////////////////
+*/
+
+/datum/symptom/weakness
+
+	name = "Weakness"
+	stealth = -3
+	resistance = -2
+	stage_speed = -2
+	transmittable = -4
+	level = 3
```

Comment:
pretty sure this determines what chem is used to roll for the symptom
from the wiki: "You can obtain symptom levels 1 and 2 with Virus Food, which you start with a dispenser full of. For symptom levels 3 and 4 you will need to acquire Unstable Mutagen. Levels 5 and 6 requires liquid plasma."
3 or 4 is probably good for this symptom